.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 .Rhistory
 .spelling  # The dictionary of words accrued by `mdspell` for spellcheck
+Rplots.pdf
```

Comment:
Depending on the outcome of https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/78#issuecomment-524402332...
* `Rplots.pdf` ends up in the analysis module itself = put this in the module specific `.gitignore` 
* `Rplots.pdf` is in the root directory = change this to `/Rplots.pdf` (see [this StackOverflow post](https://stackoverflow.com/questions/3637660/how-to-exclude-file-only-from-root-folder-in-git)) to avoid affecting other modules

Reply:
> Depending on the outcome of [#78 (comment)](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/78#issuecomment-524402332)...
> 
> * `Rplots.pdf` ends up in the analysis module itself = put this in the module specific `.gitignore`
> * `Rplots.pdf` is in the root directory = change this to `/Rplots.pdf` (see [this StackOverflow post](https://stackoverflow.com/questions/3637660/how-to-exclude-file-only-from-root-folder-in-git)) to avoid affecting other modules

`Rplots.pdf` was produced in the root directory so I made the latter change suggested above and added a comment.